charts/metallb/templates/exclude-l2-config.yaml

```
@@ -1,9 +1,12 @@
+{{- if .Values.speaker.enabled }}
```

we can use the `and` operator here.
```suggestion
{{- if .Values.speaker.enabled and .Values.speaker.excludeInterfaces.enabled }}
```

---

fixed